src/applications/appeals/996/utils/submit.js

```diff
@@ -24,17 +24,26 @@ export const getRep = formData => {
 
   // Empty string/null are not permitted values
   return removeEmptyEntries({
-    firstName: formData?.informalConferenceRep?.firstName,
-    lastName: formData?.informalConferenceRep?.lastName,
+    firstName: formData?.informalConferenceRep?.firstName.substring(
+      0,
+      MAX_LENGTH.HLR_REP_FIRST_NAME,
+    ),
+    lastName: formData?.informalConferenceRep?.lastName.substring(
+      0,
+      MAX_LENGTH.HLR_REP_LAST_NAME,
+    ),
     phone: removeEmptyEntries(phone),
-    email: formData.informalConferenceRep.email || '',
+    email: (formData.informalConferenceRep.email || '').substring(
+      0,
+      MAX_LENGTH.EMAIL,
+    ),
```

Comment:
Shouldn't we show an error on the front end instead of trimming this? Substringing an email makes it not work, right?

Reply:
The schema has a max length and it is set on the first name, last name and email inputs on the representative page - when the limit is reached inside the input, it prevents any further typing. [Try it out](https://design.va.gov/storybook/?path=/docs/uswds-va-text-input--docs) - set a `maxLength` and try to type more!

Reply:
Also, the max length on the email is 255... let's hope no one has an email string that long

Reply:
If the schema prevents it, do we need the `substring()` then? Also, I guess I need to get rid of my alternate email address... `robinisanawesomedeveloperwhocanwritejavascriptwiththebestofthemandhasbeenknowntosufferthoughpedanticandsometimesignorantpullrequestcommentsbyothermembersoftheirteambutthatsokbecausetheyhavethepatienceofasaint@some.really.obscure.mail.server.that.is.probably.a.laptop.in.a.cubicle.with.tape.on.it.that.says.do.not.touch.va.gov`

Reply:
Yeah, it isn't necessary, but I've seen weirder things happen 🤷🏽  - if you think it needs to be removed, then I will.